### PR TITLE
Adds Brion 3.3.4 release

### PR DESCRIPTION
**Pull request by NadirRoGue**

### CODE REVIEW

**Review by matz-e — APPROVED** [review]

**Comment by NadirRoGue** [comment]
retest this please

**Comment by NadirRoGue** [comment]
What is this error?

```
[2021-09-15T14:51:46.689Z] + spack-check /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/1289/config/modules.sh
[2021-09-15T14:51:46.689Z] /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/1289/spack/deploy/bin/spack-check: line 8: /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/1289/config/modules.sh: No such file or directory
script returned exit code 1
```

**Comment by matz-e** [comment]
> What is this error?
> 
> ```
> [2021-09-15T14:51:46.689Z] + spack-check /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/1289/config/modules.sh
> [2021-09-15T14:51:46.689Z] /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/1289/spack/deploy/bin/spack-check: line 8: /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/1289/config/modules.sh: No such file or directory
> script returned exit code 1
> ```

That's a red herring. Tests fail, and the config for subsequent stages is not copied over. Still seems to be a fetching issue

Retest this please

**Comment by NadirRoGue** [comment]
retest this please

**Comment by NadirRoGue** [comment]
I don't know what is making the CI tests to fail. Brion CI passes just fine, and this error does not give much information on what is failing :/

File: var/spack/repos/builtin/packages/brion/package.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ class Brion(CMakePackage):
     version('3.3.1', tag='3.3.1', submodules=True)
     version('3.3.2', tag='3.3.2', submodules=True)
     version('3.3.3', tag='3.3.3', submodules=True)
+    version('3.3.4', tag='3.3.4', submodules=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/brion/package.py:25]:
```[2021-09-22T08:59:23.543Z] Fetching tags only, you probably meant:

[2021-09-22T08:59:23.543Z]   git fetch --tags

[2021-09-22T08:59:23.543Z] error: pathspec '3.3.4' did not match any file(s) known to git.

[2021-09-22T08:59:23.543Z] ==> Error: FetchError: All fetchers failed
```
No tag yet?

Comment (NadirRoGue) [reply var/spack/repos/builtin/packages/brion/package.py:25]: No. Apparently my genius is frightening sometimes. Thanks a lot for spotting it, I just created it.

**Comment by NadirRoGue** [comment]
retest this please

**Review by matz-e — DISMISSED** [review]

**Comment by NadirRoGue** [comment]
retest this please

**Comment by matz-e** [comment]
Retest this please

**Comment by NadirRoGue** [comment]
retest this please

**Comment by NadirRoGue** [comment]
Hello @matz-e ,
I made the Brion release that was missing, but the error persists, any idea what could it be?

**Comment by matz-e** [comment]
Looks like another under-the-hood URL change in the course of moving to GitLab. I'd wait for ~2-3 hours, then rebase on top of `develop`

**Comment by NadirRoGue** [comment]
@matz-e after the rebase, the CI is still failing, should I rebase again?

**Comment by NadirRoGue** [comment]
retest this please

**Comment by matz-e** [comment]
Another rebase please, even if that just looks like a single test failure.

**Comment by NadirRoGue** [comment]
@matz-e victory! Thanks a lot!

**Review by matz-e — APPROVED** [review]

**Pull request merged**